pysmt/logics.py

```diff
@@ -590,6 +590,7 @@ def __hash__(self):
               description=\
 """Non-linear integer arithmetic with uninterpreted sort and function
 symbols.""",
+              integer_arithmetic=True,
               integer_difference=True,
```

Comment:
To check: I think we expect only one of these to be true. `integer_arithmetic` is a strict superset of `integer difference`, so I think we can remove the second. 
```suggestion
              integer_arithmetic=True,
```

Reply:
Thank you for reviewing. The following definitions of logics may be relevant to defining only one of the attributes `integer_arithmetic` and `integer_difference` as `True`:

https://github.com/pysmt/pysmt/blob/b248cdc66e8ada4a602292869cbe24dab4ea8b25/pysmt/logics.py#L457-L464

https://github.com/pysmt/pysmt/blob/b248cdc66e8ada4a602292869cbe24dab4ea8b25/pysmt/logics.py#L533-L540

I could update the pull request with a new commit with the suggested change (and the commits can be squashed upon merging), or force-push an updated commit, whichever is preferred.

I could also add a commit that defines only `integer_arithmetic=True` (and leaves `integer_difference` to the default value of 0) for the logics `QF_IDL` and `QF_UFIDL`, if it is decided relevant for this pull request to change their definitions.

Reply:
Thanks for checking. This is correct like this then. The assumption is then that we want the more specific `integer_difference`.